project/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.forms.models import ModelForm
+from tracker.settings import TIME_ZONE
+
+
+class User(AbstractUser):
+    def __repr__(self):
+        return f"<User username={self.username}>"
+
+    def __str__(self):
+        return self.username
+
+
+class habitEntry(models.Model):
```

Comment:
`HabitEntry` -- it's convention for all class names to have an initial capital letter